Model/Response/Transactions/Cancel.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
we have this file repeated in many places right?

Reply:
This is already merged in previous PR